snuba/migrations/snuba_migrations/replays/0001_replays.py

```diff
@@ -38,8 +38,8 @@
     Column("ip_address_v4", IPv4(Modifiers(nullable=True))),
     Column("ip_address_v6", IPv6(Modifiers(nullable=True))),
     # user columns
-    Column("user", String()),
-    Column("user_hash", UInt(64)),
+    Column("user", String(Modifiers(default="''"))),
```

Comment:
i'm adding this as i see it on other tables, but does a string actually need an empty string modifier? (or is it empty by default if not set).

Reply:
Yeah I can't explain why it's in some other places. Seems unnecessary to me.

Reply:
The `default` modifier is not needed. You can get rid of it. But if this column could be empty then I would recommend making it nullable.

Reply:
does this stackoverflow post apply here: https://stackoverflow.com/questions/63057886/clickhouse-string-field-disk-usage-null-vs-empty? or is null preferred?